tokio-quiche/src/quic/io/worker.rs

```diff
@@ -355,11 +362,16 @@ where
             }
 
             #[cfg(not(feature = "gcongestion"))]
-            let max_send_size = tune_max_send_size(
-                segment_size,
-                qconn.send_quantum(),
-                send_buf.len(),
-            );
+            let max_send_size = if !gcongestion_enabled {
```

Comment:
nit: since you already calculated the value you can get ride of the feature flags:

remove this from below.
```
       #[cfg(feature = "gcongestion")]
        let max_send_size = usize::MAX;
```

Reply:
Will do it as a followup.